go/oasis-test-runner/scenario/e2e/runtime/txsource.go

```diff
@@ -478,6 +478,12 @@ func (sc *txSourceImpl) Run(childEnv *env.Env) error {
 		return fmt.Errorf("WaitNodesRegistered: %w", err)
 	}
 
+	// Wait for second epoch, so that runtimes are up and running.
```

Comment:
It was always needed but this is only problematic if the default epoch time is increased so that getting to the 2nd epoch lasts more that the timeouts in the workload-runtime test (120 secs). I noticed this when I was running some tests with increased epoch time.